crates/router/src/connector/authorizedotnet.rs

```diff
@@ -142,7 +146,14 @@ impl ConnectorIntegration<api::Capture, types::PaymentsCaptureData, types::Payme
         &self,
         req: &types::PaymentsCaptureRouterData,
     ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
-        let connector_req = authorizedotnet::CancelOrCaptureTransactionRequest::try_from(req)?;
+        let connector_router_data = authorizedotnet::AuthorizedotnetRouterData::try_from((
+            &self.get_currency_unit(),
+            req.request.currency,
+            req.request.payment_amount,
```

Comment:
```suggestion
            req.request.amount_to_capture,
```
This is capture flow , amount should be capture amount